app/lib/system_process.rb

```diff
@@ -30,7 +30,11 @@ def cpu_and_proc_cycles
   end
 
   def proc_stat
-    File.read("/proc/#{pid}/stat").split if still_running?
+    return unless still_running?
+
+    # some processes put names like (Web Content) instead of (Web_Content)
+    stat = File.read("/proc/#{pid}/stat").sub(/\A.*\)/, '').split
```

Comment:
Esto está medio ruso, voy a contestar con una opción un poco más rusa:
```suggestion
    File.read("/proc/#{pid}/stat").sub(/(?<=\()(\w+(\s+)\w+)+(?=\))/, '_').split
```
Y nos ahorramos el choclo raro de abajo =)

Reply:
No quiero ni preguntar de donde sacaste eso jaja. La verdad que para el caso podría hacer `.sub(/\(.*\)/, '_').split` derecho, si no calienta lo que haya dentro del paréntesis, porque el comando lo saco de otro lado. Y queda mas simple :P

Reply:
Vos que decís?

Reply:
Dejá tu versión simplificada, la mega rusa mía quedará para la historia en los comentarios. Eso es un "look behind" y un "look forward"... hermoso para cocinar cerebros =P